pkg/traceql/expr.y

```diff
@@ -167,8 +170,9 @@ selectOperation:
   ;
 
 attribute:
-  intrinsicField    { $$ = $1 }
-  | attributeField  { $$ = $1 }
+  intrinsicField          { $$ = $1 }
```

Comment:
can we add a comment above the `intrinsicField` definition that we are no longer adding these?